modules/gapi/src/api/kernels_imgproc.cpp

```diff
@@ -115,6 +115,33 @@ cv::GArray<cv::Point2f> goodFeaturesToTrack(const GMat& image, int maxCorners, d
                                       useHarrisDetector, k);
 }
 
+GArray<GArray<Point>>
+findContours(const GMat &src, const int mode, const int method, const Point &offset)
+{
+    return imgproc::GFindContours::on(src, mode, method, offset);
+}
+
+std::tuple<GArray<GArray<Point>>,GArray<Vec4i>>
```

Comment:
Maybe there is a clever solution to this problem? Didn't you ask about that? The same functions with a different return type. And yes, it isn't overload, but m b ...

Reply:
Ok. There is no clever way.

Reply:
Unfortunately..